src/Serialization/main.jl

```diff
@@ -146,15 +146,39 @@ function metadata(;args...)
 end
 
 ################################################################################
-# Version info
+# Serialization info
 
-function get_version_info()
-  result = Dict{Symbol, Any}(
-    :Oscar => ["https://github.com/oscar-system/Oscar.jl", VERSION_NUMBER]
-  )
+function serialization_version_info(dict::Dict{Symbol, Any})
+  ns = dict[:_ns]
+  version_info = ns[:Oscar][2]
+  return version_number(version_info)
+end
+
+function version_number(v_number::String)
+  return VersionNumber(v_number)
+end
+
+# needed for older versions 
+function version_number(dict::Dict)
+  return VersionNumber(dict[:major], dict[:minor], dict[:patch])
+end
+
+function serialization_version_info()
+  if is_dev
+    path = Oscar.oscardir
+    commit_hash = readchomp(`git -C $path log -n 1 --pretty=format:"%H"`)
```

Comment:
I think we should reuse https://github.com/oscar-system/Oscar.jl/blob/master/src/utils/versioninfo.jl#L20-L35 for this. Right, @benlorenz?